integrations/docker/images/chip-cirque-device-base/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ubuntu:focal
+
+ARG OT_BR_POSIX_CHECKOUT=master
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV PLATFORM ubuntu
+ENV DOCKER 1
+ENV OT_BR_POSIX_CHECKOUT=${OT_BR_POSIX_CHECKOUT}
+
+RUN mkdir /app
+
+WORKDIR /app
+
+RUN apt-get update \
+  && apt-get install --no-install-recommends -y sudo git ca-certificates psmisc dhcpcd5 wpasupplicant wireless-tools \
+  && ln -fs /usr/share/zoneinfo/UTC /etc/localtime \
+  && git clone https://github.com/openthread/ot-br-posix . \
```

Comment:
I would separate apt-get from the git clone, so that docker layer caching works a bit better (image builds will be faster as we update what openthread we check out)

Reply:
I think we should publish the image to docker hub in the future, in that case, we should focus on the image size instead of build time.

Reply:
I would have thought image size would be the same in both cases. Is that not so?

Reply:
I think we should use https://github.com/project-chip/connectedhomeip/tree/master/third_party/ot-br-posix so that we can use the commit as third_party has